src/test/java/teammates/test/pageobjects/Browser.java

```diff
@@ -90,7 +91,19 @@ private WebDriver createWebDriver() {
                 System.out.println("Custom path: " + firefoxPath);
                 System.setProperty("webdriver.firefox.bin",firefoxPath);
             }
-            return new FirefoxDriver();
+
+            // Allow CSV files to be download automatically, without a download popup.
+            // This method is used because Selenium cannot directly interact
+            // with the download dialog.
+            // Reference:
+            // http://stackoverflow.com/questions/24852709
```

Comment:
This is tidier?
```java
// Allow CSV files to be download automatically, without a download popup.
// This method is used because Selenium cannot directly interact with the download dialog.
// Taken from http://stackoverflow.com/questions/24852709
```